src/test/credentials/provider/ssoAccessTokenProvider.test.ts

```diff
@@ -196,7 +206,7 @@ describe('SsoAccessTokenProvider', () => {
             const durationInSeconds = (endTime - startTime) / 1000
 
             //The default backoff delay is 5 seconds, the starting retry interval is 1 second
-            assert.strictEqual(durationInSeconds > 6, true, 'Duration not over 6 seconds')
+            assert.strictEqual(durationInSeconds >= 6, true, 'Duration not over 6 seconds')
```

Comment:
Is there any reason why this _wouldn't_ be 6 with the specific `tickAsync` above? If so, you should do `assert.strictEqual(durationInSeconds, 6)`.

If your value is returning a boolean, you can use `assert.ok` instead of `strictEqual`.

Reply:
Switching tickAsync to nextAsync.  This way the amount of 'time' that passes is derived from the original function.

Reply:
Does this tighten up the test tolerance at all? `>= 6` doesn't inspire confidence if you're directly controlling the clock itself.